src/Plivo/Resource/Campaign/Campaign.cs

```diff
@@ -169,6 +169,25 @@ public override string ToString()
         }
     }
 
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ImportCampaign: Resource
```

Comment:
we dont need any error msg field on this object?

Reply:
@narayana-plivo We are getting error field in case of non 2XX response without adding it as json property